addons/addon-base-raas/packages/base-raas-cfn-templates/src/templates/service-catalog/ec2-windows-instance.cfn.yml

```diff
@@ -265,15 +266,20 @@ Resources:
             KmsKeyId: !Ref EncryptionKeyArn
             DeleteOnTermination: true
       NetworkInterfaces:
-        - AssociatePublicIpAddress: true
```

Comment:
I just tried it now. When SWB tries to spin up the instance, CF fails with this error.
`Encountered unsupported boolean value AppStreamDisabled`

We want to use `AppStreamDisabled` because `AssociatePublicIpAddress` should be `true` only when  AppStream is disabled.

Reply:
I was able to get it working by writing the code like this. Which convert the boolean to a string.

`- AssociatePublicIpAddress: !If [ AppStreamEnabled, 'false', 'true' ]`